packages/devtools_app/lib/src/screens/performance/panes/timeline_events/perfetto/_perfetto_controller_web.dart

```diff
@@ -166,7 +165,7 @@ class PerfettoControllerImpl extends PerfettoController {
     );
     _initialized = true;
 
-    _perfettoIFrame = html.IFrameElement()
+    _perfettoIFrame = createElementTag('iframe') as HTMLIFrameElement
```

Comment:
Note that this cast is okay in contrast with the cast above. You're casting between interop types and you know it's that type. I *think* this isn't a lint in the future with extension types.